### PR TITLE
Unify global variables

### PR DESCRIPTION
**Pull request by osdnk**
Now all globals variables are defined in a sigle file.

Additionally, I added few things that we're using very often (basically in the same way as language syntax).

### CODE REVIEW

File: shim.js [hunk 1]
```diff
@@ -44,19 +45,14 @@ if (
...
-  get: () => ReactNative.Platform.OS === 'ios',
-  set: () => {
-    throw new Error('Trying to override internal Rainbow var');
-  },
-});
+for (let variable of Object.entries(globalVariables)) {
+  Object.defineProperty(global, variable[0], {
+    get: () => variable[1],
+    set: () => {
+      throw new Error('Trying to override internal Rainbow var');
```
Comment (brunobar79) [inline shim.js:52]: Can you add captureException from sentry here? I think that would be useful.

File: global.js [hunk 2]
```diff
@@ -1,6 +1,3 @@
-import { ENABLE_DEV_MODE } from 'react-native-dotenv';
-import { useTheme } from './src/context/ThemeContext';
-
 if (typeof btoa === 'undefined') {
   global.btoa = function (str) {
```
Comment (brunobar79) [inline global.js:5]: I assume this one can't be moved. Might be better to move it to the shim itself so we don't have 2 "global" related files

**Review by brunobar79 — COMMENTED** [review]
Added a couple of comments and also noticed that the build failed

**Comment by osdnk** [comment]
it's ready!

**Comment by brunobar79** [comment]
Build failing 😭

**Review by skylarbarrera — APPROVED** [review]
CI is 👌 , gas is just 🤑

**Pull request merged**